language-server/modules/langserver-commons/src/main/java/org/ballerinalang/langserver/commons/workspace/RunContext.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.langserver.commons.workspace;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents the context required to run a Ballerina program using the LS workspace manager.
+ *
+ * @param balSourcePath Ballerina source file path to run
+ * @param programArgs   program arguments to run the program
+ * @param env           environment variables to be added to the program
+ * @param debugPort     debug port to be used for debugging (if available)
+ * @since 2201.11.0
```

Comment:
```suggestion
 * @since 2201.12.0
```

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/43709/commits/44dccf9ff10140de11ee46b5687d48e9b6367ab9